ci-operator/step-registry/cucushift/upgrade/toimage/cucushift-upgrade-toimage-commands.sh

```diff
@@ -106,19 +106,19 @@ then
     source "${SHARED_DIR}/proxy-conf.sh"
 fi
 
-echo "RELEASE_IMAGE_INITIAL is ${RELEASE_IMAGE_INITIAL}"
 echo "RELEASE_IMAGE_LATEST is ${RELEASE_IMAGE_LATEST}"
+echo "RELEASE_IMAGE_PREVIOUS is ${RELEASE_IMAGE_PREVIOUS}"
```

Comment:
Thank you. I wonder, can we in the job configuration set 
```
 releases:
   latest:
     release:
       channel: stable
       version: "4.9"
   target:
     candidate:
       product: ocp
       stream: nightly
       version: "4.10"
```

And then use the `${RELEASE_IMAGE_TARGET}` variable? 

This PR solves the problem, but the variable naming isn't ideal

Reply:
Trying them in https://github.com/openshift/release/pull/24515, so we run the tests in parallel with this PR.